src/cli.ts

```diff
@@ -32,7 +32,9 @@ if (Deno.args.length < 1 || args.help || args.h) {
 
 const url = args._[0];
 let path = args.out;
-const authorization = args.auth;
+const authorizationString = args.auth;
+const authorizationHeader = authorizationString?.split(":")[0];
```

Comment:
Can we avoid merging this PR since it is introducing a lot of changes that will be touched again with changing the input type to a list?

Reply:
👍🏼